google_analytics/utils.py

```diff
@@ -127,6 +128,13 @@ def build_ga_params(
     # construct the gif hit url
     ga_url = "http://www.google-analytics.com/collect"
     utm_url = ga_url + "?&" + urlencode(params)
+    tracker_enabled = False
```

Comment:
To match the setting, this should be `ga_logging_enabled`

Reply:
changed